src/backend/egl/native.rs

```diff
@@ -80,6 +82,14 @@ pub struct EGLPlatform<'a> {
     _phantom: PhantomData<&'a c_void>,
 }
 
+impl<'a> Hash for EGLPlatform<'a> {
+    fn hash<H: Hasher>(&self, state: &mut H) {
+        self.platform.hash(state);
+        self.native_display.hash(state);
+        self.attrib_list.hash(state);
+    }
+}
+
```

Comment:
nit, it seems this change is no longer required

Reply:
removed